### PR TITLE
updated: round image and card shadow

### PR DESCRIPTION
**Pull request by shruthipatil11**
Please review card shadow and image round variant.

### CODE REVIEW

File: src/app/components/cards/cards.css [hunk 1]
```diff
@@ -70,9 +69,8 @@
   color: var(--GREY);
 }
 
-.card .card--shadow {
-  min-height: inherit;
-  box-shadow: 0px 0px 8px 4px rgb(0 0 0 / 5%);
+.card--shadow {
+  box-shadow: 14px 20px 20px 4px rgb(0 0 0 / 5%);
```
Comment (Maianki) [inline src/app/components/cards/cards.css]: Suggestion : You can assign rgb(0 0 0 /5%) to a CSS variable.

**Review by Maianki — APPROVED** [review]
Looks good. You have used BEM so great work there. The HTML is semantic and class names have a meaningful names. Nothing much to suggest Has added one suggestion.

File: src/app/components/cards/cards.css (hunk 1 above)
Comment (shruthipatil11) [reply src/app/components/cards/cards.css]: Thanks for review. I have made the change by substituting with css variable.

File: src/app/components/cards/cards.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .card {
-  box-shadow: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  box-shadow: var(--LIGHT-BOX-SHADOW);
   min-height: 15.2rem;
```
Comment (muskanbagrecha) [inline src/app/components/cards/cards.css]: I think you can round off that value.

**Review by muskanbagrecha — APPROVED** [review]
Small PR. Looks good.

File: src/app/components/cards/cards.css (hunk 2 above)
Comment (shruthipatil11) [reply src/app/components/cards/cards.css]: Thanks for review.
min-height is been rounded off to 15rem.I think as per our requirement we can use the decimal value for more precision.

**Pull request merged**